media-libs/smpeg2/smpeg2-2.0.0-r4.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools multilib-minimal
+
+MY_P="smpeg-${PV}"
+
+DESCRIPTION="SDL MPEG Player Library"
+HOMEPAGE="https://icculus.org/smpeg/"
+SRC_URI="https://dev.gentoo.org/~hasufell/distfiles/${MY_P}.tar.bz2"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="LGPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="debug cpu_flags_x86_mmx static-libs"
+
+DEPEND="media-libs/libsdl2[${MULTILIB_USEDEP}]"
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-gcc6.patch
+	"${FILESDIR}"/${P}-smpeg2-config.patch
+)
+
+src_prepare() {
+	default
+
+	# avoid file collision with media-libs/smpeg
+	sed -i -e '/plaympeg/d' Makefile.am || die
+
+	AT_M4DIR="${ESYSROOT}/usr/share/aclocal acinclude" eautoreconf
```

Comment:
This still kind of bother me given AT_M4DIR doesn't support paths with spaces, but ah well.

Not that it's the only thing that bother me, codebase is a mess. My wish would be to just last-rite smpeg{,2}, these are terrible and not many games need it (and sdl-{mixer,sound} can use alternatives), maybe I'll see to that another time and check if these games can work without it rather than spend more time on small things or reworking the patches.